var/spack/repos/builtin/packages/sz/package.py

```diff
@@ -30,10 +30,11 @@ class Sz(AutotoolsPackage):
     """Error-bounded Lossy Compressor for HPC Data."""
 
     homepage = "https://collab.cels.anl.gov/display/ESR/SZ"
-    url      = "https://github.com/disheng222/SZ/archive/v1.4.13.5.tar.gz"
+    url      = "https://github.com/disheng222/SZ/archive/v2.0.2.0.tar.gz"
```

Comment:
Just a minor remark: in general an update of the `url` is not needed if the "structure" of the `url` didn't change.

Reply:
I always update the URL, because OCD. But yes, it isn't necessary.